Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.10.1
+WORKDIR /go/src/github.com/netlify/twickr
+COPY . /go/src/github.com/netlify/twickr/
+RUN go get && go build
```

Comment:
For a static build, we need to add `CGO_ENABLED=0`. We should also setup glide to vendor the dependencies. I'm happy to do that if you want.

Reply:
I'll add the glide setup 👌